### PR TITLE
New source: The Property of Hate

### LINKED ISSUE #3488: [Source Request] The Property of Hate
### Source Website
- Name: The Property of Hate
- Link: http://jolleycomics.com/TPoH/
- Language(s): English

### Other Details

The structure of the web comic is split between named chapters but the numbering scheme continues to count pages.

For example, the first chapter starts with
http://jolleycomics.com/TPoH/The%20Hook/1
and ends with
http://jolleycomics.com/TPoH/The%20Hook/19

but the next chapter starts with
http://jolleycomics.com/TPoH/Rushes/20
and ends with
http://jolleycomics.com/TPoH/Rushes/38

But there is a button that links to the next page at the bottom.

### PR DESCRIPTION
**Pull request by AriaMoradi**
Will close #3488

Work In Progress for now...

### CODE REVIEW

**Comment by AriaMoradi** [comment]
The code is done, tested and it works.
review please.

File: src/en/thepropertyofhate/src/eu/kanade/tachiyomi/extension/en/thepropertyofhate/ThePropertyOfHate.kt [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+
+/**
+ *  @author Aria Moradi <aria.moradi007@gmail.com>
+ */
```
Comment (CarlosEsco) [inline src/en/thepropertyofhate/src/eu/kanade/tachiyomi/extension/en/thepropertyofhate/ThePropertyOfHate.kt:17]: remove comments

File: src/en/thepropertyofhate/src/eu/kanade/tachiyomi/extension/en/thepropertyofhate/ThePropertyOfHate.kt [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+class ThePropertyOfHate : HttpSource() {
+
+    override val name = "The Property of Hate"
```
Comment (CarlosEsco) [inline src/en/thepropertyofhate/src/eu/kanade/tachiyomi/extension/en/thepropertyofhate/ThePropertyOfHate.kt:21]: Should this be Jolley Comics?  With the chance that she makes more comics in the future?

**Review by CarlosEsco — CHANGES_REQUESTED** [review]

File: src/en/thepropertyofhate/src/eu/kanade/tachiyomi/extension/en/thepropertyofhate/ThePropertyOfHate.kt (hunk 2 above)
Comment (AriaMoradi) [reply src/en/thepropertyofhate/src/eu/kanade/tachiyomi/extension/en/thepropertyofhate/ThePropertyOfHate.kt:21]: She did, but none of the others are on this site, also this name is has more searchability

**Review by CarlosEsco — APPROVED** [review]

**Pull request merged**